content/help/ops/security/debugging-authorization/index.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Debugging Authorization
+description: Demonstrates how to debug authorization.
+weight: 5
+keywords: [debug,security,authorization,RBAC]
+---
+
+This page demonstrates how to debug Istio authorization. If you still cannot find solutions after following
+this page, feel free to send an email to `istio-security@googlegroups.com` for help.
+
+> {{< idea_icon >}}
+It would be very helpful to also include a cluster state archive in your email by following instructions in
+[reporting bugs](/help/bugs).
+
+## Ensure Authorization is Enabled Correctly
+
+The `rbacConfig` default cluster level singleton custom resource controls the authorization functionality globally.
+
+1. Run the following command to list existing `RbacConfig`:
+
+    {{< text bash >}}
+    $ kubectl get rbacconfigs.rbac.istio.io --all-namespaces
+    {{< /text >}}
+
+1. Verify there is only **one** instance of `RbacConfig` with name **default**. Otherwise, Istio disables the
```

Comment:
Consistent markup: Do not use bold for referencing code, the correct markup are backticks \`code\`
... with the `default` name ...